docs/DEVELOP.md

```diff
@@ -117,36 +184,35 @@ References:
 * Debugging dynamically loaded extensions: https://www.oreilly.com/library/view/python-cookbook/0596001673/ch16s08.html
 * Debugging Memory Problems: https://www.oreilly.com/library/view/python-cookbook/0596001673/ch16s09.html
 
-> **NOTE:** I assume gdb was already installed.
+> **NOTE:** This assumes gdb was already installed.
 
-sudo dnf install python3-debug
-sudo dnf debuginfo-install python3-debug-3.7.3-1.fc29.x86_64 #<1>
-sudo dnf debuginfo-install bzip2-libs-1.0.6-28.fc29.x86_64 glibc-2.28-27.fc29.x86_64 \
-	librsync-1.0.0-8.fc29.x86_64 libxcrypt-4.4.4-2.fc29.x86_64 \
-	openssl-libs-1.1.1b-3.fc29.x86_64 popt-1.16-15.fc29.x86_64 \
-	sssd-client-2.1.0-2.fc29.x86_64 xz-libs-5.2.4-3.fc29.x86_64 zlib-1.2.11-14.fc29.x86_64
+1. First install:
 
-1. see below for the purpose of this and the following lines
+		sudo dnf install python3-debug
+		sudo dnf debuginfo-install python3-debug-3.7.3-1.fc29.x86_64 #<1>
+		sudo dnf debuginfo-install bzip2-libs-1.0.6-28.fc29.x86_64 glibc-2.28-27.fc29.x86_64 \
+			librsync-1.0.0-8.fc29.x86_64 libxcrypt-4.4.4-2.fc29.x86_64 \
+			openssl-libs-1.1.1b-3.fc29.x86_64 popt-1.16-15.fc29.x86_64 \
+			sssd-client-2.1.0-2.fc29.x86_64 xz-libs-5.2.4-3.fc29.x86_64 zlib-1.2.11-14.fc29.x86_64
 
-Then calling:
+2. Then run:
 
-	python3 ./setup.py clean --all  #<1>
-	python3-debug ./setup.py clean --all
-	CFLAGS='-Wall -O0 -g' python3-debug ./setup.py build  #<2>
-	PATH=$PWD/build/scripts-3.7:$PATH PYTHONPATH=$PWD/build/lib.linux-x86_64-3.7-pydebug/ rdiff-backup -v 10 \
-		/some/dir1 /some/dir2
-	[...]
-	Segmentation fault (core dumped)
+		python3 ./setup.py clean --all  #<1>
```

Comment:
Remove the `#<1>` and `#<2>` - they are asciidoctor constructs and are not referred anymore in markdown.

Reply:
I did not add `#<1>` and I don't know what they are, so I'd rather not change them (and potentially garble the whole section).